CHANGELOG.md

```diff
@@ -790,6 +790,7 @@ allowing the document to be loaded. Fix: Change parent class for Formula to Text
 * make notion module discoverable
 * fix emails with `Content-Distribution: inline` and `Content-Distribution: attachment` with no filename
 * Fix email attachment filenames which had `=` in the filename itself
+* Fix documentation and sample code for Chroma
```

Comment:
please bump version. also probably should mention default/required change since that could actually break someone's workflow

Reply:
reset it so that it shouldn't break someones workflow